roles/tpa_single_node/templates/manifests/spog/api/Deployment.yaml.j2

```diff
@@ -48,26 +49,21 @@ spec:
             - -p
             - '8080'
             - --bombastic-url
-            - https://bombastic-api.trustificcation.svc.cluster.local
+            - http://bombastic-api-pod:{{ tpa_single_node_bombastic_api_port }}
             - --vexination-url
-            - https://vexination-api.trustificcation.svc.cluster.local
+            - http://vexination-api-pod:{{ tpa_single_node_vexination_api_port }}
             - --collectorist-url
-            - https://collectorist-api.trustificcation.svc.cluster.local
+            - http://collectorist-api-pod:8080
```

Comment:
@gildub @i386x this port is configurable in another PR ?

Reply:
@desmax74 , are you asking if this could be done in a separate PR ?

Reply:
@desmax74 collectorist api PR [here](https://github.com/trustification/trustification-ansible/pull/51) but I didn't made the port number configurable yet